bundles/org.openhab.ui.paperui/patch/web/js/constants.js

```diff
@@ -13,6 +13,5 @@ angular.module('PaperUI.constants', [])
   'configuration': true,
   'extensions': true,
   'rules': true,
-  'preferences': true,
-  'groups': false
```

Comment:
This was done by this commit:
https://github.com/mozilla/openhab2-addons/commit/7a041cb7bb72238e3b3b7fefa1b238cd69a3b613